components/Downloads/DownloadHeader/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { FormattedMessage, injectIntl } from 'react-intl';
+import styles from './index.module.scss';
+import SectionTitle from '../../Common/SectionTitle';
+import type { WrappedComponentProps } from 'react-intl';
+import type { NodeReleaseData } from '../../../types';
+
+type Props = {
```

Comment:
```suggestion
type DownloadHeaderProps = { release: NodeReleaseData }
```

The `release` shouldn't be optional